services/spar/src/Spar/Intra/Brig.hs

```diff
@@ -69,15 +90,15 @@ instance MonadSparToBrig m => MonadSparToBrig (ReaderT r m) where
 
 
 -- | Create a user on brig.
-createUser
+createBrigUser
```

Comment:
We still need to clarify whether we prefer to do something like this; or whether we want to just qualify all our uses;

This version works better with my setup and passes the duplicate ID linter; but we should probably pick one and stick with it. This commit is currently easily revertable